Workbooks/UsageMetrics/Usage - Users/Usage - Users.workbook

```diff
@@ -535,6 +558,93 @@
       },
       "name": "query - 10"
     },
+    {
+      "type": 1,
+      "content": {
+        "json": "#### Last month\r\n"
+      },
+      "conditionalVisibility": {
+        "parameterName": "IsGreaterOrEqual30days",
+        "comparison": "isEqualTo",
+        "value": "true"
+      },
+      "name": "text - 16"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": " let now = now();\r\n    let previousMonthTimestamp = now - 60d;\r\n    let monthTimestamp = now - 30d;\r\n    let mainTable = union pageViews,customEvents\r\n    | where timestamp >= previousMonthTimestamp\r\n    ;\r\n    \r\n     let cohortedTable = mainTable   \r\n    | extend timestamp = timestamp  - 420m;\r\n    let thisMonth = toscalar(mainTable\r\n    | where timestamp >= monthTimestamp\r\n    | summarize hll = hll(user_Id));\r\n    let lastMonth = toscalar(mainTable\r\n    | where timestamp < monthTimestamp and timestamp >= previousMonthTimestamp\r\n    | summarize hll = hll(user_Id));\r\n    range i from 1 to 1 step 1\r\n    | extend ThisMonth = thisMonth, LastMonth = lastMonth\r\n    | evaluate dcount_intersect(LastMonth, ThisMonth)\r\n    | project Mau = dcount_hll(ThisMonth), MauLastMonth = s0, ReturningUsers = s1\r\n    | extend NewUsers = Mau - ReturningUsers, LostUsers = MauLastMonth - ReturningUsers, GrowthRate = round(100.0 * (Mau - MauLastMonth) / MauLastMonth), RetentionRate = round(100.0 * ReturningUsers / MauLastMonth)\r\n    | project Mau, RetentionRate = iff(MauLastMonth ==0, '--', strcat(tolong(RetentionRate),'%')), GrowthRate = iff(MauLastMonth ==0, '--', strcat(tolong(GrowthRate),'%'))\r\n    | evaluate narrow()\r\n    | project Column, Value, id = case(Column =='Mau', 'Community-Workbooks/Usage/Active Users', Column =='RetentionRate', 'Community-Workbooks/Retention/Retention - N Period','Community-Workbooks/Usage/New, Returning and Churned Users')\r\n    | where Value != '--'\r\n\r\n \r\n",
+        "size": 3,
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components",
+        "visualization": "tiles",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "Mau",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "RetentionRate",
```

Comment:
Should this be "Retention Rate"? And should all of these titles be in all caps to match the cards above them?

Reply:
(Same goes for "GrowthRate"->"Growth Rate")

Reply:
Changed